### PR TITLE
new:usr:Add instructions to use Rubix with Apache Hive

### PR DESCRIPTION
**Pull request by vrajat**
Add documentation to use Rubix with Apache Hive on AWS EMR. The install guide has
been restructured to separate install instructions for each engine.

### CODE REVIEW

**Comment by vrajat** [comment]
@GodenYao can you please review this PR ?

File: docs/install/cluster.md [hunk 1]
```diff
@@ -0,0 +1,18 @@
+# Start and connect to master
+
+## Start an EMR Cluster (Optional)
+
+Create EMR cluster, use 'advanced option' to create a cluster with the required spec.
```
Comment (GodenYao) [inline docs/install/cluster.md]: "Create **an** EMR cluster..."

File: docs/install/cluster.md [hunk 2]
```diff
@@ -0,0 +1,18 @@
...
+1. Specify atleast 30GB space for Root device EBS volume.
```
Comment (GodenYao) [inline docs/install/cluster.md]: missing space between "atleast"

File: docs/install/index.rst [hunk 3]
```diff
@@ -0,0 +1,23 @@
+.. _install-index:
+
+##################
+Installation Guide
+##################
+
+
+This section provides instructions to install Rubix and use it with Presto or
+Hive. Instructions are tested on an EMR 5.X cluster. Same instructions should work
+with installations on-prem and other distributions. Please `contact <https://groups.google.com/forum/#!forum/rubix-users>`_
```
Comment (GodenYao) [inline docs/install/index.rst]: what do you mean by "other distributions" here?  other distribution of Presto /Hive version? 
What are the versions we tested in this guide?

File: docs/install/cluster.md [hunk 4]
```diff
@@ -0,0 +1,18 @@
+# Start and connect to master
+
+## Start an EMR Cluster (Optional)
```
Comment (GodenYao) [inline docs/install/cluster.md]: probably want to add a few words why EMR is optional any alternatives, etc.

File: docs/install/query.md [hunk 5]
```diff
@@ -0,0 +1,41 @@
...
+## Create External Table
+
+    CREATE EXTERNAL TABLE wikistats_orc_rubix 
+    (language STRING, page_title STRING,
+    hits BIGINT, retrived_size BIGINT)
+    STORED AS ORC
+    LOCATION 'rubix://emr.presto.airpal/wikistats/orc';
+
+## Run Query (Presto or Hive CLI)
+
```
Comment (GodenYao) [inline docs/install/query.md]: I assume the syntax below is correct in both Presto and Hive.

File: docs/install/cluster.md [hunk 6]
```diff
@@ -0,0 +1,18 @@
...
+
+Create EMR cluster, use 'advanced option' to create a cluster with the required spec.
+1. Specify atleast 30GB space for Root device EBS volume.
+2. To login into cluster, choose a EC2 key pair.
+
+## Connect to Master Node
+
+Log into master and all slave nodes as "hadoop" user.
```
Comment (GodenYao) [inline docs/install/cluster.md]: slave -> worker?

File: docs/install/hive.md [hunk 7]
```diff
@@ -0,0 +1,33 @@
...
+    hive --service metastore --start
+
+## Configure Apache Hive to use Rubix FileSystem
+
+    hive --hiveconf \
+	    fs.rubix.impl=com.qubole.rubix.hadoop2.CachingNativeS3FileSystem \
+	    fs.rubix.awsAccessKeyId=<AWS ACCESS KEY> \
+	    fs.rubix.awsSecretAccessKey=<AWS SECRET ACCESS KEY>
+
+## (Advanced) Configure Apache Hive to use Rubix FileSystem for S3 and S3A schemes
```
Comment (GodenYao) [inline docs/install/hive.md]: maybe link to some documentation to explain anything special for s3

File: docs/install/hive.md [hunk 8]
```diff
@@ -0,0 +1,33 @@
...
+    add jar /usr/lib/rubix/lib/rubix-bookkeeper.jar
+    add jar /usr/lib/rubix/lib/rubix-core.jar
+    add jar /usr/lib/rubix/lib/rubix-hadoop2.jar
+
+## Restart Hive Metastore Server
+
+    hive --service metastore --stop
+    hive --service metastore --start
+
+## Configure Apache Hive to use Rubix FileSystem
```
Comment (GodenYao) [inline docs/install/hive.md]: do you want to keep consistence "Rubi**X**" ?

File: docs/install/hive.md (hunk 7 above)
Comment (GodenYao) [inline docs/install/hive.md]: do you want to keep consistence "Rubi**X**" ?

File: docs/install/query.md [hunk 9]
```diff
@@ -0,0 +1,41 @@
...
+    FROM default.wikistats_orc_rubix
+    WHERE language = 'en'
+    AND page_title NOT IN ('Main_Page',  '404_error/')
+    AND page_title NOT LIKE '%index%'
+    AND page_title NOT LIKE '%Search%'
+    GROUP BY language, page_title
+    ORDER BY avg_hits DESC
+    LIMIT 10;
+
+## Rubix Stats (supported on Presto only)
```
Comment (GodenYao) [inline docs/install/query.md]: do you want to keep consistence "Rubi**X**" ?

File: docs/install/index.rst (hunk 3 above)
Comment (vrajat) [reply docs/install/index.rst]: I have clarified that it is tested on EMR 5.x only.

File: docs/install/cluster.md (hunk 4 above)
Comment (vrajat) [reply docs/install/cluster.md]: Removed optional

File: docs/install/query.md (hunk 5 above)
Comment (vrajat) [reply docs/install/query.md]: Yes. Verified

File: docs/install/hive.md (hunk 7 above)
Comment (vrajat) [reply docs/install/hive.md]: I explained the effect of this configuration

File: docs/install/hive.md (hunk 8 above)
Comment (vrajat) [reply docs/install/hive.md]: Done

File: docs/install/hive.md (hunk 7 above)
Comment (vrajat) [reply docs/install/hive.md]: Done

File: docs/install/query.md (hunk 9 above)
Comment (vrajat) [reply docs/install/query.md]: Done

**Pull request merged**